### DISCUSSION TITLE
Where is the option to configure with other VPN apps?

### DISCUSSION THREAD

**Discussion by shuvashish76** [celzero/rethink-app#1650] [Q&A] [answered]
https://github.com/celzero/rethink-app/blob/main/fastlane/metadata/android/en-US/phoneScreenshots/3.png

Can't find this option. Where is this specific settings?

**Comment by ignoramous** [comment, marked as answer]
The [screenshot shows](https://github.com/celzero/rethink-app/blob/main/fastlane/metadata/android/en-US/phoneScreenshots/3.png) WireGuard configurations imported into Rethink form Proton's and Mullvad's websites.

Also: https://github.com/celzero/rethink-app/discussions/940

**Reply by shuvashish76** [reply]
Thanks. CMIIW so currently it's not possible to use [Bitmask](https://f-droid.org/packages/se.leap.bitmaskclient/)/[Riseup](https://f-droid.org/packages/se.leap.riseupvpn/)/[Calyx](https://f-droid.org/packages/org.calyxinstitute.vpn/) VPN _(both the client source code and the server side setup are foss_) with Rethink?

https://leap.se
> LEAP VPN code has been third party audited by Cure53 and is fast, reliable, and easy to use.

Requested upstream : https://gitlab.com/CalyxOS/calyxos/-/issues/2635

**Reply by ignoramous** [reply]
LEAP does not support WireGuard. Their client apps could support SOCKS5 (like you've requested them), too, which is also easier to implement.
